tests/integration/defs/accuracy/test_cli.py

```diff
@@ -14,16 +14,18 @@
 # limitations under the License.
 import pytest
 
+from tensorrt_llm.llmapi import (EagleDecodingConfig, LookaheadDecodingConfig,
+                                 MedusaDecodingConfig)
 from tensorrt_llm.quantization import QuantAlgo
 
 from ..conftest import (llm_models_root, skip_no_nvls, skip_pre_ada,
                         skip_pre_blackwell, skip_pre_hopper)
-from .accuracy_core import (AccuracyTestHarness, CnnDailymail, Humaneval, Mmlu,
-                            PassKeyRetrieval64k, PassKeyRetrieval128k,
+from .accuracy_core import (CliAccuracyTestHarness, CnnDailymail, Humaneval,
```

Comment:
It takes me some time to understand `test_cli.py` means that convert -> build -> run flow. But I don't find a better name yet.